sample-code/src/server.ts

```diff
@@ -51,6 +53,100 @@ app.get('/azure-openai/chat-completion', async (req, res) => {
   }
 });
 
+app.get('/azure-openai/chat-completion-stream', async (req, res) => {
+  const controller = new AbortController();
+  try {
+    const response = await chatCompletionStream(controller);
+
+    res.setHeader('Cache-Control', 'no-cache');
+    res.setHeader('Content-Type', 'text/event-stream');
+    res.setHeader('Access-Control-Allow-Origin', '*');
+    res.setHeader('Connection', 'keep-alive');
+    res.flushHeaders();
+
+    let connectionAlive = true;
+
+    res.on('close', () => {
+      controller.abort();
+      connectionAlive = false;
+      res.end();
+    });
+
+    for await (const chunk of response.stream.toContentStream()) {
+      if (!connectionAlive) {
+        break;
+      }
+      res.write(chunk);
+    }
```

Comment:
[pp] seems to be some reusable code :)

Reply:
I removed the multiple choice example. I will mention this feature in the documentation later